### PR TITLE
Add some necessary changes 

### PR DESCRIPTION
**Pull request by ragu-1997**
rm -rf is the command unfortunately missed the -rf in previous commit so, sometime file is not removed. safer side rm -rf is necessary  
Shell file final exit code is necessary (exit 0). we miss the exit code some time it is running the background

### CODE REVIEW

**Comment by jubalh** [comment]
I saw now that you edited the text here on the GitHub comment.

Please squash all commits together into one and add the explanation there as well. Like mentioned in the contributing.md. 

About your comments:
i don't see what you mean about `rm -rf` you don't change anything in this regard.

Also i think you didn't explain the nohup and wget changes.

File: .builds/openbsd.yml [hunk 1]
```diff
@@ -22,6 +22,9 @@ packages:
   - libgcrypt
   - libsignal-protocol-c
   - sqlite3
+  - wget
+  - nohup
```
Comment (sjaeckel) [inline .builds/openbsd.yml:26]: Where is this file even used?

If you change it, make it used in CI, otherwise simply remove it IMO

**Review by sjaeckel — CHANGES_REQUESTED** [review]
The three `exit 0` are borderline nonsense, I'd say they are even harmful.

The default shell behavior is that the exit code of a shell script is equal to the exit code of the last executed process/command. Those changes make these scripts simply always succeed, whereas before their exit code was appropriately the exit code of the thing that should be done.

**Comment by ragu-1997** [comment]
Sorry for the misunderstanding Thanks for your explanation sir

**Pull request closed without merging**